helper-cli/src/main/kotlin/commands/MergeRepositoryConfigurationsCommand.kt

```diff
@@ -40,7 +40,7 @@ internal class MergeRepositoryConfigurationsCommand : CliktCommand(
     private val inputRepositoryConfigurationFiles by option(
         "--input-repository-configuration-files", "-i",
         help = "A comma separated list of the repository configuration files to be merged."
-    ).convert { it.split(",").toList().map { File(it.expandTilde()) } }
+    ).convert { csv -> csv.split(",").toList().map { File(it.expandTilde()) } }
```

Comment:
How about fixing this by applying a simplification like `.convert { File(it.expandTilde()) }.split(",").required()`?

Reply:
BTW, if you feel like it, similar simplification could be done in `ListLicensesCommand` and `ListPackagesCommand` where you also call `split()` "manually".

Reply:
Good idea! Done.

Reply:
Maybe I should make a break, but is it just me or did the original proposal not work due to reversed order of split and convert?

Reply:
I didn't test it, but IMO it should work, as we're doing something similar e.g. at https://github.com/oss-review-toolkit/ort/blob/90d12ee0c008967581fbca2a41e15a8a9744427c/cli/src/main/kotlin/commands/AnalyzerCommand.kt#L76

I.e. with *clikt* the order is "reversed" in than [you first specify the conversion, and then the split operation](https://ajalt.github.io/clikt/api/clikt/com.github.ajalt.clikt.parameters.options/split/). This seems counter-intuitive at first, but the `split()` doesn't really split immediately, but records a transformation to be performed later.